README.md

```diff
@@ -1 +1,22 @@
-# Myfirst_Porfolio
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+**Portfolio**
+
```

Comment:
- [ ] Please consider updating your readme file adding more information and properly aligning sections. Currently readme looks very confusing:

Your README  | An example of a professional README
------------- | -------------
<img width="447" alt="Screenshot 2022-03-17 at 20 45 43" src="https://user-images.githubusercontent.com/77038610/158851213-ad12ebfe-3fa4-451b-9750-94d74fcfe1b2.png">  | <img width="609" alt="Screenshot 2022-03-17 at 20 45 31" src="https://user-images.githubusercontent.com/77038610/158851176-0a84e776-787b-439d-90ee-13e918fb20fb.png">

[Here](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) you can read more and learn about formatting `.md` documents on Github